changelog/11797.bugfix.rst

```diff
@@ -0,0 +1 @@
+Ensure approx for SequenceLike objects doesn't wrap those sequences in a scalar.
```

Comment:
```suggestion
Ensure :func:`pytest.approx` for :class:`Sequence <collections.abc.Sequence>`-like objects does not wrap those sequences in a scalar.
```

Or:

```suggestion
:func:`pytest.approx` now correctly handles :class:`Sequence <collections.abc.Sequence>`-like objects.
```

But then this feels more like an improvement than a bugfix? 🤔

Reply:
I'll recategorize and rebase as soon as I get to it again